### ISSUE TITLE
Falsely sortavalues, e.g. data-sortvalue="0" cause wrong sorting

### ISSUE THREAD

**Issue by koosvanderkolk** [myspace-nu/jquery.fancyTable#54]
```
var cmpa = $(elma).data('sortvalue') ? $(elma).data('sortvalue') : elma.html();
var cmpb = $(elmb).data('sortvalue') ? $(elmb).data('sortvalue') : elmb.html();
```
In case sortvalue equals 0, -1, etc. above code will take the html instead.

**Comment by koosvanderkolk** [comment]
```
var cmpa = typeof $(elma).data('sortvalue') !== 'undefined' ? $(elma).data('sortvalue') : elma.html();
var cmpb = typeof $(elmb).data('sortvalue') !== 'undefined' ? $(elmb).data('sortvalue') : elmb.html();
```
seems to fix it.

(sorry for not adding pull requests: I forked your code for my own project)

**Comment by myspace-nu** [comment]
Fixed in 10b5b471e279e4ac7e58fe2458817137dc1b0fa2
